docs/content/docs/connectors/datastream/kafka.md

```diff
@@ -213,26 +213,115 @@ Note that Kafka source does **NOT** rely on committed offsets for fault toleranc
 is only for exposing the progress of consumer and consuming group for monitoring.
 
 ### Monitoring
-Kafka source exposes metrics in Flink's metric group for monitoring and diagnosing.
-#### Scope of Metric
-All metrics of Kafka source reader are registered under group ```KafkaSourceReader```, which is a 
-child group of operator metric group. Metrics related to a specific topic partition will be registered
-in the group ```KafkaSourceReader.topic.<topic_name>.partition.<partition_id>```.
 
-For example, current consuming offset of topic "my-topic" and partition 1 will be reported in metric: 
-```<some_parent_groups>.operator.KafkaSourceReader.topic.my-topic.partition.1.currentOffset``` ,
+Kafka source exposes the following metrics in the respective [scope]({{< ref "docs/ops/metrics" >}}/#scope).
 
-and number of successful commits will be reported in metric:
-```<some_parent_groups>.operator.KafkaSourceReader.commitsSucceeded``` .
+#### Scope of Metric
 
-#### List of Metrics
+All metrics of Kafka source reader are registered under group ```<job metric group>.<operator name>.<subtask>.KafkaSourceReader```, which is a 
```

Comment:
And which metrics belong to the Kafka source reader?

Reply:
hm this is a good point. I actually don't know why we have the metric group `KafkaSourceReader` and don't just add it directly to the source subtask. WDYT, wouldn't that feel more natural?

Reply:
organizing kafka-specific metrics into their own group is fine imo, but KafkaSourceReader is quite specific isn't it.
If these metrics existed before 1.14 we shouldn't change things to to not break existing setups.

Reply:
Unfortunately, these changes have already been backported to 1.13. So I leave them as is.

Reply:
Considering that they were not present in 1.13.0/1.13.1 it _may_ be fine to change things?
Why do things like this get casually backported in a bugfix release -.-